scripts.js

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env node
+'use strict';
+/* eslint-disable no-undef */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const {spawn} = require('child_process');
+require('dotenv').config();
+
+const rawArgs = process.argv.slice(2);
+
+function parseArgs(numFixedArgs, expectedOptions) {
+  const fixedArgs = [];
+  const options = {};
+  const extra = [];
+  const alreadyCounted = {};
+  for (let i = 0; i < rawArgs.length; i++) {
+    const rawArg = rawArgs[i];
+    if (rawArg.startsWith('--')) {
+      const optionName = rawArg.slice(2);
+      const optionDetected = expectedOptions[optionName];
+      if (!alreadyCounted[optionName] && optionDetected) {
+        alreadyCounted[optionName] = true;
+        if (optionDetected === 'boolean') {
+          options[optionName] = true;
+        } else {
+          i++;
+          options[optionName] = rawArgs[i];
+        }
+      } else {
+        if (fixedArgs.length < numFixedArgs) {
+          throw new Error(
+            `expected ${numFixedArgs} fied agrs, got only ${fixedArgs.length}`
```

Comment:
just spot a missing an x at fied (fixed)

Reply:
also, `agrs` should be `args` :)